### PR TITLE
implemented task and tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: src/createUrl.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+'use strict';
```
Comment (roman-mate) [inline src/createUrl.test.js]: let's cover invalid input too?

File: src/createUrl.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const createUrl = require('./createUrl');
+
+test('Create URL with 2 params', () => {
+  expect(createUrl(
+    'http://localhost:3000/api/countries/{country}/regions/{region}/',
+    { country: 'Ukraine', region: 'Kiev' }
```
[the remark this answers is not in the document]
Comment (ivadimko) [reply src/createUrl.test.js]: what a shame :D

[the remark this answers is not in the document]
Comment (ivadimko) [reply src/createUrl.test.js]: btw it exists now https://github.com/mate-academy/fe-program/blob/master/js/tasks/create-url.js

[the remark this answers is not in the document]
Comment (roman-mate) [reply src/createUrl.test.js]: yeah! @mgrinko looking at you :D

[the remark this answers is not in the document]
Comment (mgrinko) [reply src/createUrl.test.js]: @roman-mate 
Yes you are right
I also did not noticed it when copied a task from Boris's lesson :)

File: src/createUrl.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/**
+ * Implement createUrl function:
+ *
+ * Function takes string (template) and object (params). It returns URL based on
+ * template and replaces all `{name}` with params.name.
+ *
+ * createUrl('/api/{list}/{id}', {list: 'public/items', id: 0}) ===
+ *    '/api/public/items/0'
```
Comment (mgrinko) [inline src/createUrl.js]: this example is a bit tricky
I would put it the last one

File: src/createUrl.test.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    '/api/{user}/{params}/{social}',
+    { user: 'Petya', params: 'public/params', social: 'facebook' }
+  ))
+    .toBe('/api/Petya/public/params/facebook');
+});
+
+test('Create URL with 5 params', () => {
+  expect(createUrl(
+    '/api/{a}/{b}/{c}/{d}/{e}', { a: 1, b: 2, c: 3, d: 4, e: 5 }
+  ))
```
Comment (mgrinko) [inline src/createUrl.test.js]: I think we should test for `undefined` too if param is not given

**Pull request merged**